config/src/main/resources/org/springframework/security/config/spring-security-5.3.rnc

```diff
@@ -572,6 +572,48 @@ provider.attlist &=
 	## The URI used to discover the configuration information for an OAuth 2.0 or OpenID Connect 1.0 Provider.
 	attribute issuer-uri {xsd:token}?
 
+oauth2-resource-server =
+	## Configures authentication support as an OAuth 2.0 Resource Server.
+	element oauth2-resource-server {oauth2-resource-server.attlist}
```

Comment:
I'm confused how `<jwt>` and `<opaque-token>` are `<xs:choice>`'s in the `xsd`, however, both elements are not defined as choices in the `rnc` file? It looks like both those elements do not have `<oauth2-resource-server>` as their parent. Maybe I'm missing something?

It should be similar to `websocket-message-broker`:

```xml
websocket-message-broker =
	element websocket-message-broker { websocket-message-broker.attrlist, (intercept-message* & expression-handler?) }
```


```(intercept-message* & expression-handler?)``` are choices